packages/vapor-demo/components/Svg.js

```diff
@@ -1,9 +1,27 @@
 import * as VaporSVG from 'coveo-styleguide';
 
-export const Svg = ({name, ...props}) => {
+export const Svg = ({name, svgClass = '', ...props}) => {
     if (VaporSVG.svg[name]) {
+
+        const withSvgClass = (svgString, svgClass) => {
```

Comment:
This function could be defined outside the Svg component so it does not get redified on each render and for each SVG

Reply:
Done